classes/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_accessor :first_name, :last_name, :items
```

Comment:
Great job so far 👏 , can you consider quickly removing the items instance variable from attr_accessor since in the UML diagram it is defined as a private instance variable.

Use instead
`attr_reader: items`